### ISSUE TITLE
push back the deadline

### ISSUE THREAD

**Issue by ntfshard** [PocketSprite/CodingChallenge#6]
Hello

May be you can push back the deadline to allow submitters complete their projects. It's a narrow window between receiving a device and the deadline.

**Comment by Spritetm** [comment]
Yeah, seeing as the amount of entries at the moment is... well, we can't even give away all prizes, I've pushed the deadline back three months.

**Comment by ishotjr** [comment]
@Spritetm much appreciated!! :bowtie: :blue_heart:

**Comment by rorosaurus** [comment]
@Spritetm wiki edit is missing the month :) https://github.com/PocketSprite/CodingChallenge/wiki/PocketSprite-Coding-Challenge/_compare/d89dce0%5E...d89dce0

**Comment by dulsi** [comment]
So what is the deadline? Assuming three months that means November. I have a game I've been making for the Pocket Arcade called colormonster (https://github.com/dulsi/colormonster). Unfortunately it looks like it can't directly work on PocketSprite due to the loss of 8 pixels on the two side.

**Comment by ishotjr** [comment]
@Spritetm can we get some clarification on the deadline?  The last edit, on September 3rd, updated it to say:

> Deadline closes 31, 2018 - we will announce the winner of the PocketSprite Programming Competition shortly after.

There are only two months between the edit and the end of the year that have 31 days: October and December; which is it?  Thanks!! :bowtie: 

https://github.com/PocketSprite/CodingChallenge/wiki/PocketSprite-Coding-Challenge/_compare/994c92236bf59da57aee31cc4dce7f334260251f...d89dce0db64af87b6a54ee211b3c7802a251032b
